pkg/metrics/metrics.go

```diff
@@ -92,6 +92,21 @@ var (
 	},
 		[]string{"outcome"})
 
+	// EndpointRegenerationTime is the total time taken to regenerate endpoint
+	EndpointRegenerationTime = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: Namespace,
+		Name:      "endpoint_regeneration_seconds",
+		Help:      "Total sum of successful endpoint regeneration times",
```

Comment:
Nit: "Total sum of endpoint regeneration times" sounds odd to me; why not "time taken to regenerate endpoint" as per the documentation of this counter?

Reply:
This change was part of one of the review suggestions... It was originally `time taken to ....` and was asked to be changed to this...

Reply:
For sure, it's not a blocking issue for me.